README.md

```diff
@@ -70,6 +70,11 @@ brew install gremlins
 Download the appropriate release package from the [release page](https://github.com/go-gremlins/gremlins/releases/latest), 
 extract the zip archive and copy the `.exe` file somewhere in your `PATH`.
 
+#### Docker 
+
```

Comment:
You can add a sentence like: "You can also run Gremlins using the official Docker image: ..."